src/components/CurrencyInputPanel/SwapCurrencyInputPanel.tsx

```diff
@@ -255,6 +256,7 @@ export default function SwapCurrencyInputPanel({
         </FixedContainer>
       )}
       <Container hideInput={hideInput}>
+        <ThemedText.SubHeaderSmall color="textTertiary">{label}</ThemedText.SubHeaderSmall>
```

Comment:
```suggestion
        {Boolean(label) && <ThemedText.SubHeaderSmall color="textTertiary">{label}</ThemedText.SubHeaderSmall>}
```

Reply:
label should always be defined, it's no longer optional